### PR TITLE
Expose plays, JobEvent

### PR DESCRIPTION
**Pull request by syncrou**
1. Expose `job_plays` off of `AnsibleTowerClient::Job`
2. Add `job_events` as a Collection
3. Add `AnsibleTowerClient::JobEvent` 

Testing Notes

1. Lookup a Job
`job = connection.api.jobs.find n`

2. Call job_plays off of said job
`job.job_plays.first`

3. Returned object will be a `AnsibleTowerClient::JobEvent`

### CODE REVIEW

**Comment by syncrou** [comment]
@miq-bot assign @gmcculloug 

@miq-bot add_label enhancement

cc - @bzwei

**Comment by gmcculloug** [comment]
@bdunne Please review.

File: spec/job_spec.rb [hunk 1]
```diff
@@ -37,6 +37,21 @@
     end
   end
 
+  context '#job_plays' do
+    let(:url)            { "example.com/api/v1/job_plays" }
+    let(:job_collection) { build(:response_collection, :klass => described_class) }
+    let(:job_events)     { build(:response_url_collection, :klass => AnsibleTowerClient::JobEvent, :url => url) }
+    it "returns a collection of AnsibleTowerClient::JobEvents" do
```
Comment (bzwei) [inline spec/job_spec.rb]: Why `job_plays` returns a collection of `JobEvents`? Shouldn't them be `JobPlays`?

Comment (syncrou) [inline spec/job_spec.rb]: @bzwei - Yes - it should be - but interestingly enough - I'm getting job_events back - so I'm digging into why.  

I was thinking the endpoint /jobs/5/job_plays/2 would actually return a JobEvent - it isn't - the related field points to job_events.  Trying to see why our Gem is returning a different type.

Comment (syncrou) [inline spec/job_spec.rb]: @bzwei -Not sure how we should handle this - The Gem is doing the right thing - `job_plays` return data of `type` = `job_event` Here is a sample directly from the Tower API:

```
GET /api/v1/jobs/56/job_plays/
HTTP 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept
X-API-Time: 0.040s




{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "play": "Hello World Sample",
            "failed_count": 0,
            "changed_count": 0,
            "started": "2016-11-29T21:34:42.175Z",
            "changed": false,
            "related": {
                "job_event": "/api/v1/job_events/151/"
            },
            "skipped_count": 0,
            "failed": false,
            "ok_count": 2,
            "unreachable_count": 0,
            "type": "job_event",
            "id": 151
        }
    ]
}
```

Comment (bzwei) [inline spec/job_spec.rb]: Why did you say "The Gem is doing the right thing"? Your example shows it is job_play, not job_event.

Comment (syncrou) [inline spec/job_spec.rb]: With the current discussion in regards to a `JobPlay` class for results off of the `/job_plays` endpoint - I suggest we merge this as is and I will build a new PR to address the type override.

Comment (bdunne) [inline spec/job_spec.rb]: :+1: Sounds good.

**Pull request merged**